### PR TITLE
Updates to help/moderation

### PR DESCRIPTION
**Pull request by jweiskoff**
General updates to help/moderation after discussion amongst stakeholders.

### CODE REVIEW

File: help/moderation.md [hunk 1]
```diff
@@ -1,68 +1,98 @@
...
-arXiv is a forum for professional members of the scientific community, providing rapid distribution of new research and maintaining a permanent collection of scientific work. As part of the service all submissions are subject to moderation. For more general information about the moderation process please [see our 2019 blog post](http://blog.arxiv.org/2019/08/29/our-moderation-process/).
-
+arXiv is a forum for professional members of the scientific community, providing rapid distribution of new research and maintaining a permanent collection of scientific work. As part of the service all submissions are subject to moderation. For more general information about the moderation process please [see our 2019 blog post](https://blog.arxiv.org/2019/08/29/our-moderation-process/).
+
+- [arXiv moderation](#arxiv-moderation)
+  - [Why does arXiv moderate submissions?](#why-moderate) 
+  - [What policies guide moderation before public announcement?](#what-policies)
+      - [Reclassification of works](#reclassification)
+      - [Declined Submissions](#declined)
+      - [Scholarly interest](#scolarly-interest)
```
Comment (jimentwood) [inline help/moderation.md:10]:
```suggestion
      - [Scholarly interest](#scholarly-interest)

```

**Review by afromme — APPROVED** [review]

File: help/moderation.md [hunk 2]
```diff
@@ -1,68 +1,98 @@
...
+Submissions in a format deemed inappropriate may be rejected. Such as abstract-only submissions, presentations, book announcements, book reviews, submissions without references, calls for papers, articles containing advertisements or links to other repositories may be rejected. We do not accept submissions with line numbers or submissions with watermarks. Germane "Comment on" and "Reply to" articles must still conform to normal scholarly discourse.  arXiv limits these submissions to a single "reply to" and "comment on" per original paper, and updates to the discourse can be made through [replacements](/help/replace).
```
Comment (jimentwood) [inline help/moderation.md:52]:
```suggestion
Submissions in a format deemed inappropriate, such as abstract-only submissions, presentations, book announcements, book reviews, submissions without references, calls for papers, articles containing advertisements or links to other repositories, may be rejected. We do not accept submissions with line numbers or submissions with watermarks. Germane "Comment on" and "Reply to" articles must still conform to normal scholarly discourse.  arXiv limits these submissions to a single "reply to" and "comment on" per original paper, and updates to the discourse can be made through [replacements](/help/replace). 

```

**Review by jimentwood — CHANGES_REQUESTED** [review]
2 minor suggested changes

**Comment by jweiskoff** [comment]
I think I resolved those two as you were going.

**Review by jimentwood — APPROVED** [review]
looks good

**Review by afromme — APPROVED** [review]
looks good

**Pull request merged**